contents/docs/data/cohorts.mdx

```diff
@@ -78,7 +80,7 @@ If your only goal is to create a **list of users** with something in common, we
 
 ## How to create a cohort
 
-There are two methods to create a new cohort:
+[Person profiles](/docs/data/persons) are required for creating cohorts. Once you've captured some person profiles, there are two ways to create a new cohort:
```

Comment:
I feel like this clarification is unnecessary given the note about profiles being needed for cohorts above.